pipeline/MUSIC_DATA.csv

```diff
@@ -0,0 +1,19 @@
+,item_type,item_description,album_title,artist_name,country,amount_paid_usd,genres,release_date,sale_date
+0,t,Johanna in the 80s,None,Sweeney,Australia,6.5,"['adelaide', 'alternative', 'australian', 'electronic pop', 'queerpop', 'synthpop', 'Australia']",02-12-2024,03-12-2024
+1,a,Steve Rothery Band - Live in Frankfurt,None,Steve Rothery,United Kingdom,8.87,"['progressive rock', 'rock', 'Aylesbury']",30-11-2024,03-12-2024
+2,a,UFO 50,None,Eirik Suhrke,United States,10.0,"['chiptune', 'soundtrack', 'video game music', 'jazz', 'synth', 'Norway']",02-10-2024,03-12-2024
```

Comment:
Seems to be no title for some data - I think we should discuss in next meeting how might be best to handle missing data